lib/api/controllers/remoteActions/swagger.js

```diff
@@ -44,58 +43,69 @@ generateSwagger = (routes, apiVersion) => {
 
   routes.forEach(route => {
     var 
-      reg = /:([^\/]*)/g, 
+      reg = /:([^\/]*)/g, // since url parameters got the following form "/:parameterName/:otherParameter" this reg captures, all chars following : which are not a / char
+                          // the first and las / are reg delimiters
```

Comment:
nitpicking: well... the first comment line is **very** helpful, but the others are only explaining regular expressions, and can/should be removed